cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/DatasetModulesDeployer.java

```diff
@@ -38,32 +40,31 @@
 /**
  * Deploys Dataset Modules.
  */
-public class DatasetModulesDeployer {
+final class DatasetModulesDeployer {
   private static final Logger LOG = LoggerFactory.getLogger(DatasetModulesDeployer.class);
   private final CConfiguration cConf;
   private final DatasetFramework datasetFramework;
   // An instance of InMemoryDatasetFramework is used to check if a dataset is a system dataset
   private final DatasetFramework systemDatasetFramework;
-  private final Id.Namespace namespace;
   private final boolean allowDatasetUncheckedUpgrade;
 
-  public DatasetModulesDeployer(DatasetFramework datasetFramework, DatasetFramework inMemoryDatasetFramework,
-                                Id.Namespace namespace, CConfiguration cConf) {
+  DatasetModulesDeployer(DatasetFramework datasetFramework, DatasetFramework
+    inMemoryDatasetFramework, CConfiguration cConf) {
```

Comment:
nit: this is a weird line break, inMemoryDatasetFramework can be on the previous line or entirely on this line